README.md

```diff
@@ -24,3 +24,10 @@ Things you may want to cover:
 * Deployment instructions
 
 * ...
+
+## Cheat sheet
+```bash
+# generates the spec directory 
+rails generate spec:install
```

Comment:
What is this for?
If it's only a one-time thing and then it will be configured in the app, I wouldn't add it to the README.

Reply:
Ok. I will delete it.

Reply:
The section is already deleted.